### PR TITLE
Fix PT `DeepPot` and replace ASE calculator

### PR DESCRIPTION
**Pull request by njzjz**
- Set `deepmd.pt.utils.ase_calc.DPCalculator` as an alias of `deepmd.calculator.DP`;
- Replace `deepmd_pt` with  `deepmd.pt` in `deep_pot.py`; fix (atomic) virial output shape of `DeepPot`; add tests for them;
- Set `pbc` in `pt/test_calculator.py` as it requests stress.

### CODE REVIEW

**Comment by njzjz** [comment]
> Set pbc in pt/test_calculator.py as it requests stress

I think it's meaningless to calculate stress for a non-periodic system, right?

File: deepmd/infer/deep_pot.py [hunk 1]
```diff
@@ -56,7 +56,7 @@ def __new__(cls, model_file: str, *args, **kwargs):
 
                 return super().__new__(DeepPotTF)
             elif backend == DPBackend.PyTorch:
-                from deepmd_pt.infer.deep_eval import DeepPot as DeepPotPT
+                from deepmd.pt.infer.deep_eval import DeepPot as DeepPotPT
```
Comment (wanghan-iapcm) [inline deepmd/infer/deep_pot.py:59]: I am wondering why it is not detected by a UT.

**Comment by wanghan-iapcm** [comment]
> > Set pbc in pt/test_calculator.py as it requests stress
> 
> I think it's meaningless to calculate stress for a non-periodic system, right?

no it does not make sense

File: deepmd/infer/deep_pot.py (hunk 1 above)
Comment (wanghan-iapcm) [reply deepmd/infer/deep_pot.py:59]: We may want to check with the pt backend in the UTs of python inference

**Review by wanghan-iapcm — COMMENTED** [review]
please add the UT as commented.

**Review by wanghan-iapcm — APPROVED** [review]

**Pull request merged**